x-pack/plugins/lens/public/plugin.ts

```diff
@@ -156,6 +161,12 @@ export class LensPlugin {
   start(core: CoreStart, startDependencies: LensPluginStartDependencies) {
     this.attributeService = getLensAttributeService(core, startDependencies);
     this.createEditorFrame = this.editorFrameService.start(core, startDependencies).createInstance;
+    // unregisters the Visualize action and registers the lens one
+    startDependencies.uiActions.unregisterAction(ACTION_VISUALIZE_FIELD);
```

Comment:
`unregisterAction` will throw if there is no `ACTION_VISUALIZE_FIELD` action. 
Is this a possible scenario? could visualise be disabled independent from lens? 
it is possible to check if action exists first with `hasAction`

Reply:
Thanx for that @Dosant, not sure if this is a possible scenario but better safe than sorry. Will add this :)